### PR TITLE
[101] - Querying the Transportation Database

### PR DESCRIPTION
**Pull request by MYKEL7**
## Task: `Querying the Transportation Database`
- `I queried the transportation database`

### CODE REVIEW

File: submissions/onyemeMichael/database/transportationQuery.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/** Total number of passengers who suvrvived **/
+SELECT COUNT(survived) FROM accidents WHERE survived = 1;
+/** answer: 269 **/
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportationQuery.sql:3]: Your answer is incorrect

I guess you didn't import your data correctly. To even confirm, the total number of records on the CSV file for survival sheet is 891. If your answer is correct when you sum the two results you have, it ought to give you that total (i.e 891). But 269 + 419 = 688 instead

File: submissions/onyemeMichael/database/transportationQuery.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+/** Total number of passengers who didn not survive **/
+SELECT COUNT(survived) FROM accidents WHERE survived = 0;
+/** answer: 419 **/
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportationQuery.sql:7]: Your answer is incorrect

File: submissions/onyemeMichael/database/transportationQuery.sql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+WHERE age < 27 
+AND point_of_embarkation != 'S';
+
+/** number of passengers who survived and point of embarkation was from Southampton **/
+SELECT COUNT(*) 
+FROM accidents
+LEFT JOIN trips 
+ON accidents.passenger_id = trips.passenger_id 
+WHERE trips.point_of_embarkation = 'S' 
+AND accidents.survived = 1;
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportationQuery.sql:23]: no answer here

File: submissions/onyemeMichael/database/transportationQuery.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+AND trips.number_of_siblings_spouses > 0 
+AND passengers.age > 35; 
+
+/** the total number of passengers who paid a fare greater than $100 and above the age of 35 with siblings or spouses on board **/
+SELECT COUNT(*) FROM passengers
+LEFT JOIN trips
+ON passengers.id = trips.passenger_id
+WHERE trip_fare > 100
+AND age > 35 
+AND number_of_siblings_spouses  != 0;
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportationQuery.sql:40]: no answer here too

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly clear all the records in your tables, import the CSV file again and rerun your query. We are all using the same imported data so all result should be the same

File: submissions/onyemeMichael/database/transportationQuery.sql (hunk 1 above)
Comment (MYKEL7) [reply submissions/onyemeMichael/database/transportationQuery.sql:3]: The csv file for survival sheet which we download is 688.

survival_import

Comment (adaugochi) [reply submissions/onyemeMichael/database/transportationQuery.sql:3]: check the sheet and see the total number of data there

Comment (adaugochi) [reply submissions/onyemeMichael/database/transportationQuery.sql:3]: image

Comment (adaugochi) [reply submissions/onyemeMichael/database/transportationQuery.sql:3]: Remember we are both importing the same record, I will adviced you DROP your table, create the table again and import the data gain.

**Comment by MYKEL7** [comment]
I have drop the table, I even drop the database and started all over, and
I'm till having the same result.
I tried setting sql_mode =" "; till yet the same number.

**Comment by adaugochi** [comment]
@MYKEL7 well done

Please kindly reach out to other scholars that experience the same issue @Samju200, @faithakinbobola and Benja they are likely to help with the issue

**Review by adaugochi — APPROVED** [review]

**Pull request merged**